src-vue/src/components/PullRequestsSelector.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+    <div>
+        <el-collapse @change="onChange">
```

Comment:
Ok turns out removing the `accordion` attribute breaks calling `getPullRequests()`? Wtf?

Didn't happen when testing before previous review cause it had already loaded the list of PRs and the hot-reloaded when I removed the attribute.

Running 6ab48c0f3ec21fb8844a8a18b26146546ae1ea4a from cold start I cannot get the PR list to load for neither Mods nor Launcher. Adding a debug `console.log()` at the beginning of `getPullRequests()` I can tell that the function is never called x_x

Reply:
Confirmed fixed in 27a0e892dcaf6a88a93f8679993854c8a915a99e